Tests/test_PDB.py

```diff
@@ -741,89 +820,96 @@ def test_strict(self):
+            "O CB CG CD1 CD2 N CA C O CB CG1 CG2 N CA C O CB "
+            "CG CD OE1 NE2 N CA C O CB CG OD1 ND2 N CA C O CB "
+            "N CA C O CB CG OD1 ND2 N CA C O CB CG CD N CA C "
+            "O CB CG OD1 OD2 N CA C O CB SG N CA C O CB CG CD "
+            "CE NZ N CA C O CB OG1 CG2 N CA C O CB CG1 CG2 "
+            "CD1 N CA C O CB CG CD1 CD2 N CA C O CB CG CD CE "
+            "NZ N CA C O CB N CA C O CB CG CD1 CD2 N CA C O N "
+            "CA C O CB CG CD N CA C O N CA C O CB N CA C O CB "
+            "OG1 CG2 N CA C O CB CG CD1 CD2 N CA C O CB CG CD "
+            "OE1 OE2 N CA C O CB CG CD OE1 OE2 N CA C O CB CG "
+            "SE CE N CA C O CB CG SE CE N CA C O CB OG1 CG2 N "
+            "CA C O CB N CA C O CB SG N CA C O CB CG CD OE1 "
+            "NE2 N CA C O OXT O O O O O O O O O O O O O O O O "
+            "O O O O O O O O O O O O O O O O O O O O O O O O "
+            "O O O O O O O O O O O O O O O O O O O O O O O O "
+            "O O O O O O O O O O O O O O O O O O O O O O O O",
+        )
+        self.assertEqual(
+            " ".join(atom.element for atom in chain.get_atoms()),
+            "N C C O C C SE C N C C O C C O O N C C O C C C C "
+            "N C C O C C C N C N N N C C O C C C O N N C C O "
+            "N C C O C C C N C C O C C C C N N C C O C C C O "
+            "O N C C O C C C N C C O C C C C C C C N C C O C "
+            "C C N C N N N C C O C C O O N C C O C C C C C C "
+            "C O N C C O C C C N C C O C C O O N C C O C C C "
+            "N C N N N C C O C C C C C C C N C C O C C C C C "
+            "C C O N C C O C C C C N N C C O C O C N C C O C "
+            "C C C N C C O C C C N C N N N C C O C N C C O C "
+            "C C O O N C C O C C C O N N C C O C N C C O C O "
+            "N C C O C C C O N N C C O C C C O O N C C O C C "
+            "C N C C O C C C C N N C C O C C O N N C C O C C "
+            "C C N C C C C C N C C O C C SE C N C C O C O C N "
+            "C C O C C C O O N C C O C O C N C C O C C C C N "
+            "C C O C C C C N C C O C C C N C C O C C C O N N "
+            "C C O C C O N N C C O C N C C O C C O N N C C O "
+            "C C C N C C O C C O O N C C O C S N C C O C C C "
+            "C N N C C O C O C N C C O C C C C N C C O C C C "
+            "C N C C O C C C C N N C C O C N C C O C C C C N "
+            "C C O N C C O C C C N C C O N C C O C N C C O C "
+            "O C N C C O C C C C N C C O C C C O O N C C O C "
+            "C C O O N C C O C C SE C N C C O C C SE C N C C "
+            "O C O C N C C O C N C C O C S N C C O C C C O N "
+            "N C C O O O O O O O O O O O O O O O O O O O O O "
+            "O O O O O O O O O O O O O O O O O O O O O O O O "
+            "O O O O O O O O O O O O O O O O O O O O O O O O "
+            "O O O O O O O O O O O O O O O O O O O O O",
+        )
 
     def test_model_numbering(self):
         """Preserve model serial numbers during I/O."""
+
```

Comment:
This blank line is a problem:

```
$ flake8
./Tests/test_PDB.py:911:1: D202 No blank lines allowed after function docstring
```

For now we've putting a comment line here to avoid the ``D202`` versus ``black`` disagreement.